### PR TITLE
Milestone 10

### PR DESCRIPTION
**Pull request by levy002**
My portfolio milestone 10.

  In this milestone 10, We:
- Added projects popup markup dynamically by using Javascript
-  Styled the popups and make them responsive.

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      description: ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe."],
+      technologies: ['html', 'Ruby on rails', 'css', 'Github'],
+      seeLiveLink: '#',
+      seeSourceLink: '#',
+    },
+  },
+];
```
Comment (rustam-tolipov) [inline popup.js]: ## WELL DONE 👏😁

- [x] Your popup is working well and also responsive. Please kindly create objects for each project. Currently, you have just one object, if you fix this problem you should be approved by the next reviewer ✅. **_GOOD LUCK 🍀_**

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: popup.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    technologies: ['html', 'Ruby on rails', 'css', 'Github'],
+    seeLiveLink: '#',
+    seeSourceLink: '#',
+  },
+];
+
+const AllButtons = document.getElementsByClassName('see-project');
```
Comment (moise10r) [inline popup.js:36]: - [ OPTIONAL] Kindly, use lower case notation or UPPERCASE your Constance variable  name [JavaScript const](https://www.javascripttutorial.net/es6/javascript-const/#:~:text=By%20convention%2C%20the%20constant%20identifiers,keyword%20can't%20be%20reassigned.&text=However%2C%20variables%20created%20by%20the%20const%20keyword%20are%20%E2%80%9Cimmutable%E2%80%9D.)

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @levy002** , 

### **Congratulations! 🎉 Project Approved ✅**

Great work on making the changes requested by the previous reviewer
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 


- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.✔️  
- When the user clicks (or taps) the close (X) button, the popup disappears.
- The popup is responsive, it fits mobile and desktop sizes properly. keep rocking :rocket: 


Cheers and Happy coding!👏👏👏💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**